client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -986,6 +996,29 @@ class WpcomChecklistComponent extends PureComponent {
 			/>
 		);
 	};
+
+	renderSiteMenuUpdateTask = ( TaskComponent, baseProps, task ) => {
+		const { translate } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				bannerImageSrc="/calypso/images/stats/tasks/personalize-your-site.svg"
+				completedButtonText={ translate( 'Edit' ) }
+				completedDescription={ translate( 'You can edit your site menu whenever you like.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **16** times:
`translate( 'You can edit your site title whenever you like.' )` *ES Score: 7*